content/docs/gke/private-clusters.md

```diff
@@ -331,8 +330,20 @@ export PROJECT_NUMBER=$(gcloud projects describe kubeflow-dev --format='value(pr
 1. Update iap-ingress component parameters:
 
     ```
-    cd ${KFAPP}/ks_app
-    ks param set iap-ingress privateGKECluster true
+    cd ${KFAPP}/kustomize
+    gvim basic-auth-ingress.yaml  # Or iap-ingress.yaml if you are using IAP
+    ```
+
+   Find and set the `privateGKECluster` parameter to true:
```

Comment:
Apply indentation to fix list formatting: 
https://deploy-preview-938--competent-brattain-de2d6d.netlify.com/docs/gke/private-clusters/